Atomic/client.py

```diff
@@ -41,20 +48,26 @@ def __getattr__(self, name):
 
     def __getattribute__(self, name):
         # Avoid recursion for self._dockerclient
-        if name == "_dockerclient":
+        if name == "_dockerclient" or name == "_client":
             return object.__getattribute__(self, name)
         obj = self._dockerclient
         attr = docker.AutoVersionClient.__getattribute__(obj, name)
         if hasattr(attr, '__call__'):
             def newfunc(*args, **kwargs):
-                result = attr(*args, **kwargs)
-                return iter_subs(result)
+                try:
+                    result = attr(*args, **kwargs)
+                    return iter_subs(result)
+                except requests.exceptions.ConnectionError as e:
```

Comment:
The goal of these changes is to do 

atomic images list
atomic ps --all

Both of these look for docker images as system containers, and potentially others in the future.  We don't want to catch all calls to self.dockerclient.images() and self.dockerclient.containers(), this little change allows us to not messy up the code when the docker service is not running.